s2a/src/test/java/io/grpc/s2a/internal/handshaker/FakeWriter.java

```diff
@@ -79,6 +71,18 @@ enum VerificationResult {
   private String failureReason;
   private PrivateKey privateKey;
 
+  public static String convertInputStreamToString(InputStream is) throws IOException {
```

Comment:
You could use guava CharStreams instead of rolling your own: `CharStreams.toString(new InputStreamReader(
       inputStream, Charsets.UTF_8))`

Reply:
Done in https://github.com/grpc/grpc-java/pull/11702/commits/3c2e0ead8e42f956cf9d9dd9ab5909538d7e5257